account.go

```diff
@@ -1,22 +1,260 @@
 package stripe
 
-import "encoding/json"
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+)
+
+type LegalEntityType string
+
+type IdentityVerificationStatus string
+
+type Interval string
+
+const (
+	Individual  LegalEntityType = "individual"
```

Comment:
These are actually only 'company' and 'individual' (we accept more, but that's all we're exposing)